analysis/README.md

```diff
@@ -6,17 +6,17 @@ This crate is where most of the compilation pipeline lives. It takes a parse tre
 
 We define two types of machines: virtual machines and constrained machines. Constrained machines are the lower level kind of machine. They have a notion of blocks through a latch and an operation_id. Virtual machines are a higher level type of machines. For each machine type, we provide the elements which appear in that type.
 
-|                         | Virtual | Constrained |
```

Comment:
Could you please undo these changes? There are also some newly introduced or removed newlines below.